src/troubleshooting.ts

```diff
@@ -59,6 +93,17 @@ export class TroubleshootingProvider implements WebviewViewProvider {
       .map((webviewUri) => `<link href="${webviewUri}" rel="stylesheet">`)
       .join("\n");
 
+    let hljsUri;
+    if (window.activeColorTheme.kind === ColorThemeKind.Light) {
+      hljsUri = this._view.webview.asWebviewUri(
+        Uri.joinPath(this._extensionUri, "media", "github.min.css")
+      );
+    } else {
+      hljsUri = this._view.webview.asWebviewUri(
+        Uri.joinPath(this._extensionUri, "media", "github-dark.min.css")
+      );
+    }
+
```

Comment:
it would be nice to share this rendering and markdown code with the chat

Reply:
Agree